iib/workers/tasks/opm_operations.py

```diff
@@ -190,3 +191,201 @@ def _serve_cmd_at_port(serve_cmd, cwd, port, max_tries, wait_time):
         rpc_proc.kill()
 
     raise IIBError(f'Index registry has not been initialized after {max_tries} tries')
+
+
+def opm_migrate(index_db, base_dir):
+    """
+    Migrate SQLite database to File-Based catalog using opm command.
+
+    :param str index_db: path to SQLite index.db which should migrated to FBC.
+    :param str base_dir: base directory to which the database file should be saved.
+    :return: Returns path to directory containing file-based catalog.
+    :rtype: str
+    """
+    from iib.workers.tasks.utils import run_cmd
+
+    fbc_dir_path = os.path.join(base_dir, 'catalog')
+
+    if os.path.exists(fbc_dir_path):
+        shutil.rmtree(fbc_dir_path)
+
+    cmd = ['opm', 'migrate', index_db, fbc_dir_path]
+
+    run_cmd(cmd, {'cwd': base_dir}, exc_msg='Failed to migrate index.db to file-based catalog')
+    log.info("Migration to file-based catalog was completed.")
+    return fbc_dir_path
+
+
+def omp_generate_dockerfile(fbc_dir, base_dir, index_db, binary_image=None, dockerfile_name=None):
```

Comment:
Should this be `opm_generate_dockerfile`?

Reply:
:facepalm: